src/style.css

```diff
@@ -1,3 +1,89 @@
+html {
+  margin: 0;
+}
+
 body {
+  margin: 0 ;
+}
+
+.header {
+  display: flex;
+  flex-direction: row-reverse;
+  align-items: center;
+  max-width: 1200px;
+  background: rgba(255, 255, 255, 1);
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  padding: 0 40px;
+}
+
+.logo {
+  order: 1;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  top: 10px;
+  left: 50px;
+
+}
+
+.nav {
+  margin: 0;
+}
+
+.nav-list {
+  height: 58px;
+  list-style: none;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
   margin: 0;
+  padding: 0;
+}
+
+.link {
+  padding: 23px 0;
+  margin: 0 10px;
+  flex-shrink: 0;
+  align-self: center;
+  text-decoration: none;
+  position: relative;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  text-transform: uppercase;
+  font-weight: 500;
+  color: rgba(0, 0, 0, 1);
+  line-height: normal;
+}
+
+@media (min-width: 1024px) {
+  .nav {
+    left: 333px;
+  }
+}
+
+@media (min-width: 1200px) {
+  .nav {
+    left: 509px;
+  }
+}
+
+.link:hover {
+  color: rgba(0, 172, 220, 1);
+}
+
+.is-active {
+  color: rgba(0, 172, 220, 1);
```

Comment:
Move all repeatable colors to the variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)